hera_mc/cm_revisions.py

```diff
@@ -82,6 +84,7 @@ def get_last_revision(hpn, session=None):
     session:  db session
     """
 
+    rq = 'LAS'
```

Comment:
I think it would be better to spell out the full words (LAST, ACTIVE, etc) here and below, to make things more understandable for new people who are reading the code.